applications/luci-app-sshtunnel/htdocs/luci-static/resources/view/sshtunnel/ssh_servers.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+'require form';
+'require fs';
+'require uci';
+'require ui';
+'require view';
+
+var allSshKeys = {};
+
+return view.extend({
+	load: function () {
+		return L.resolveDefault(fs.list('/root/.ssh/'), []).then(function (entries) {
+			var tasks = [];
+			for (var i = 0; i < entries.length; i++) {
+				if (entries[i].type === 'file' && entries[i].name.match(/\.pub$/)) {
+					tasks.push(Promise.resolve(entries[i].name));
+				}
+			}
+			return Promise.all(tasks);
+		});
+	},
+
+	render: function (data) {
+		var sshKeys = _splitSshKeys(data);
+		if (sshKeys.length === 0) {
+			ui.addNotification(null, E('p', _('No SSH keys found, <a href="./ssh_keys">generate a new one</a>')), 'warning');
+		}
+
+		var m, s, o;
+
+		m = new form.Map('sshtunnel', _('SSH Tunnels'),
+			_('This configures <a href="https://openwrt.org/docs/guide-user/services/ssh/sshtunnel">SSH Tunnels</a>.')
+		);
+
+		s = m.section(form.GridSection, 'server', _('Servers'));
+		s.anonymous = false;
+		s.addremove = true;
+		s.nodescriptions = true;
+
+		o = s.tab('general', _('General Settings'));
+		o = s.tab('advanced', _('Advanced Settings'));
+
+		o = s.taboption('general', form.Value, 'hostname', _('Hostname'), '');
+		o.placeholder = 'example.com';
+		o.datatype = 'host';
+		o.rmempty = false;
+
+		o = s.taboption('general', form.Value, 'port', _('Port'), '');
+		o.placeholder = '22';
+		o.datatype = 'port';
+
+		o = s.taboption('general', form.Value, 'user', _('User'), '');
+		o.default = 'root';
+
+		o = s.taboption('general', form.ListValue, 'IdentityFile', _('Key file'),
+			_('Private key file with authentication identity. ' +
+				'See <em>ssh_config IdentityFile</em>')
```

Comment:
please also change `em` to `code` here

Reply:
Did this also get updated? Ensure all files get the `<em>` to `<code>` treatment